test/dogma/rules/trailing_whitespace_test.exs

```diff
@@ -29,6 +29,26 @@ defmodule Dogma.Rules.TrailingWhitespaceTest do
     ]
   end
 
+  with "windows line endings are okay" do
+    setup context do
+      source = "   'hello'\r\n"
+      <> "'how'\r\n"
+      <> "  'are'\r\n"
+      <> "      'you?'\r\n"
+      <> "'WINDOWS!' \r\n"
+      script = source |> Script.parse( "foo.ex" ) |> TrailingWhitespace.test
```

Comment:
I'll have a look at sorting these tests out later today. I think you're right it should be clearer why that one is bad but the others are okay.